perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/AttributesManagerMethod.java

```diff
@@ -1275,16 +1275,42 @@ public Void call(ApiCaller ac, Deserializer parms) throws PerunException {
 	 * @param attribute AttributeDefinition object
 	 * @return AttributeDefinition Created AttributeDefinition
 	 */
+	/*#
+	 * Creates AttributeDefinition
```

Comment:
Especially for attribute it would be nice to have example param values or specify allowed values in docs explicitly. I'm not sure its documented anywhere and sadly it's no represented as enum either. 

This goes for ``type`` param. Also stating format for ``namespace`` and ``friendlyName`` would be nice.